x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_add_edit/fields/script_recorder_fields.tsx

```diff
@@ -0,0 +1,133 @@
+/*
```

Comment:
Pulled over from legacy uptime, with a few adjustments to make it work with react-hook-form.